src/js/06-tabs-block.js

```diff
@@ -115,18 +126,20 @@ document.addEventListener('DOMContentLoaded', function () {
 
       // Build an array of elements
       var elements = []
+      var showSingle = false
 
       // add sections for each language from driver manual html output format
       langList.forEach(function (lang) {
         tab.querySelectorAll('.include-with-' + lang).forEach(function (block) {
           block.setAttribute('data-title', lang)
           block.setAttribute('data-lang', lang)
           elements.push(block)
+          if (gdsModes.includes(lang)) showSingle = true
         })
       })
```

Comment:
I think it would be better to iterate over `langList` and `gdsModes`. It does not feel right to add `data-lang=mutate`.